design/account.go

```diff
@@ -170,6 +170,7 @@ var userDataAttributes = a.Type("UserDataAttributes", func() {
 	a.Attribute("url", d.String, "The url")
 	a.Attribute("company", d.String, "The company")
 	a.Attribute("providerType", d.String, "The IDP provided this identity")
+	a.Attribute("cluster", d.String, "The cluster where the user has her OpenShift account is created")
```

Comment:
Let's maybe more specific here what cluster means here. With examples. We will need to refer to the OpenShift API URL as a cluster I guess.

Reply:
Updated this to be more descriptive: 7a9539d